Directory/Directory.java

```diff
@@ -0,0 +1,30 @@
+package tat.bsu.dir;
+
+import java.io.*;
+
+/**
+ * Create HTML page that has information about current Directory:
+ * name of file , type , creation date and size
+ */
+public class Directory {
+    public static final String absolutePackagePath = "C:\\Documents and Settings\\Admin\\IdeaProjects\\Directory\\src\\tat\\bsu\\dir\\";
```

Comment:
That violates specification - it must be current directory, the directory in which you run application, not hard-coded path. Specification violation is not acceptable